### PR TITLE
electrum: update to 3.1.3.

### PR DESCRIPTION
**Pull request by cr6git**
Electrum is a lightweight Bitcoin wallet, see [Documentation](http://docs.electrum.org/en/latest/index.html).

### CODE REVIEW

File: srcpkgs/electrum/template [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+short_desc="Lightweight Bitcoin wallet"
 maintainer="Charles E. Lehner <cel@celehner.com>"
 homepage="https://electrum.org/"
 license="MIT"
 distfiles="https://download.electrum.org/${version}/Electrum-${version}.tar.gz"
-checksum=3195679f898bd763c6db32f46bbdf59fa5a211fd3013507f5c7aab25d9b94337
+checksum=f5a67c616984e18646a6369e38d203e49e371ad904e6556ff445110f5ab44559
 
 post_install() {
 	vlicense LICENCE LICENSE
```
Comment (maxice8) [inline srcpkgs/electrum/template]: lol

File: srcpkgs/electrum/template [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 nocross=yes
 wrksrc="Electrum-${version}"
 build_style=python3-module
 pycompile_module="electrum"
 hostmakedepends="python3-pip"
-depends="python3-dnspython python3-pyaes python3-ecdsa python3-qrcode
- python3-pbkdf2 python3-protobuf python3-pysocks python3-PyQt5
- python3-jsonrpclib python3-requests"
+depends="python3-dnspython python3-ecdsa python3-jsonrpclib
+ python3-pbkdf2 python3-protobuf python3-pyaes python3-pysocks
```
Comment (maxice8) [inline srcpkgs/electrum/template]: I have the suspicion that `xtrimdeps` can help with this.

Comment (cr6git) [reply srcpkgs/electrum/template]: I don't know, how to install & use `xtrimdeps` ?

Comment (maxice8) [reply srcpkgs/electrum/template]: available on https://github.com/maxice8/xtools-devel , just put `xtrimdeps` and `xgetdeps` to your path

Comment (cr6git) [reply srcpkgs/electrum/template]: it's okay, I installed it. `xtrimdeps srcpkgs/electrum/template` returns nothing.

**Comment by asergi** [comment]
@cr6git why you closed? If you resubmit it, I'll start reviewing it.

**Comment by cr6git** [comment]
@asergi it's okay, I re-opened https://github.com/voidlinux/void-packages/pull/15162

**Pull request closed without merging**